SCClassLibrary/Common/GUI/PlusGUI/Core/HistoryGui.sc

```diff
@@ -66,6 +68,8 @@ HistoryGui : JITGui {
 				};
 			};
 		})
+		.font_(font)
+		.tabWidth_(font.size * 0.6 * 4) // ca. 4 chars
```

Comment:
Maybe better: 
`"1234".bounds(font).width`?

Reply:
`tabWidth_` seems to be a pixel size, according to documentation

Reply:
yes, that is better, done.